pkg/pillar/scripts/device-steps.sh

```diff
@@ -130,6 +130,15 @@ else
         mkdir $PERSISTDIR/vault
     fi
 fi
+# XXX FIXME: this is an "Indiana Jones snatch" attempt at
+# swapping content and snapshotter folders undeneath containerd.
+# Just like th original Indiana Jones's move this is highly risky.
+for i in io.containerd.snapshotter.v1.overlayfs io.containerd.content.v1.content; do
```

Comment:
This might work, or it might get stuck because of open file descriptors in containerd. I am not sure.

Reply:
Even if we do this, we should do it as:

```sh
for i in $PERSISTDIR/containerd/*containerd.snapshotter.*; do
```

so we aren't hard-coding specific ones in.

Either way... ouch.